src/xcode/ENA/ENA/Source/Models/Home/HomeRiskCellConfigurator/HomeRiskCellConfigurators/HomeRiskLevelCellConfigurator.swift

```diff
@@ -109,7 +108,24 @@ class HomeRiskLevelCellConfigurator: HomeRiskCellConfigurator {
 		cell.bodyLabel.accessibilityIdentifier = AccessibilityIdentifiers.RiskCollectionViewCell.bodyLabel
 		cell.detectionIntervalLabel.accessibilityIdentifier = AccessibilityIdentifiers.RiskCollectionViewCell.detectionIntervalLabel
 		cell.updateButton.accessibilityIdentifier = AccessibilityIdentifiers.RiskCollectionViewCell.updateButton
+	}
+
+	// MARK: Hashable
+
+	func hash(into hasher: inout Swift.Hasher) {
+		hasher.combine(isLoading)
```

Comment:
super not there

Reply:
for this case, `isLoading` suddenly must be combined into the hasher. How come?

Reply:
all properties that the class depends on must be included